.travis.yml

```diff
@@ -34,6 +34,8 @@ branches:
 env:
   global:
     - COVERALLS_PARALLEL=true
+    # Ensure Xdebug v3 coverage reporting is available.
+    - XDEBUG_MODE=coverage
```

Comment:
This ensures that coverage reporting is enabled for Xdebug v3 shipped with Travis CI.